ros2/content.md

```diff
@@ -0,0 +1,265 @@
+# What is [ROS 2](https://index.ros.org/doc/ros2)?
+
+The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications. From drivers to state-of-the-art algorithms, and with powerful developer tools, ROS has what you need for your next robotics project. And it's all open source. ROS 2 is the next-generation ROS: building upon modern middleware for decentralized discovery, message serialization, and secure transport with quality of service enabling high performance, low latency, language agnostic distributed computing for robotic systems.
+
+> [wikipedia.org/wiki/Robot_Operating_System](https://en.wikipedia.org/wiki/Robot_Operating_System)
+
+[%%LOGO%%](https://index.ros.org/doc/ros2)
```

Comment:
I pulled the new logo from the ROS2 github org gravatar, but perhaps there is a different logo we'd like to use for branding?

Reply:
If we plan on merging with the current `ros` repo, then I'll just go back to the version agnostic logo.